golem_messages/message/tasks.py

```diff
@@ -353,3 +335,24 @@ class SubtaskPaymentRequest(base.Message):
     verify_key='report_computed_task',
     verify_class=ReportComputedTask,
 )
+
+def deserialize_force_get_task_result_failed(key, value):
```

Comment:
Why not move this to `message.concents` as I mentioned? :)

Reply:
ah, oops...

Reply:
@jiivan moved